src/wp-admin/admin.php

```diff
@@ -33,6 +33,12 @@
 
 require_once dirname( __DIR__ ) . '/wp-load.php';
 
+//Redirect to the admin dashboard when /admin,php is accessed without any query parameters.
+if ( strpos( $_SERVER['REQUEST_URI'], 'wp-admin/admin.php' ) !== false && empty( $_SERVER['QUERY_STRING'] ) ) {
```

Comment:
```suggestion
if ( str_contains( $_SERVER['REQUEST_URI'], 'wp-admin/admin.php' ) && empty( $_SERVER['QUERY_STRING'] ) ) {
```

Reply:
Hi @mukeshpanchal27, won't "str_contains" cause a deprecation error if the PHP version is less than 8?